### DISCUSSION TITLE
Getting a Rectangle as an image from a PDF

### DISCUSSION THREAD

**Discussion by Aadil-5122** [pymupdf/PyMuPDF#2207] [Looking for help] [answered]
I just want help in the above specified issue that:

I want a particular part (a Specified Rectangle) of PDF as image (using Python and PyMuPDF) only!
is there any way out ?

**Comment by JorjMcKie** [comment, marked as answer]
If you know the rectangle coordinates on a page, simply do this:
```python
pix = page.get_pixmap(clip=rect)
pix.save("your-image.png")
```
Please remember that `get_pixmap()` has a number of parameters, that let you choose the desired colorspace, resolution (dpi) and more.

**Comment by Aadil-5122** [comment]
Thanks alot sir!
